frontend/src/app/_components/creator-card/creator-card.component.html

```diff
@@ -0,0 +1,10 @@
+<div  class="creator card">
+    <img [routerLink]="'/' + component + '/' + creator.id" src="{{creator.images[0].url}}" class="card-img-top" alt="{{creator.images[0].alt}}" />
```

Comment:
Why not make it a div with the routerLink and make the img and card-body be children of that div?

Reply:
Oh, or even better, make it an <a> anchor with the routerLink and display:block in the css so that it'll set the cursor correctly?

Reply:
it made all the text underlined - also it made the bottom block very confusing. will totally take practical solutions to those problems that aren't this though :)